test/local_integration_test.py

```diff
@@ -125,6 +137,20 @@ def check_bdbag(self, response: bytes):
                     self.assertTrue(num_rows > 1)
                     logger.info(f'BDBag file {file_name} contains {num_rows} rows.')
 
+    def download_file_from_drs_response(self, response: bytes):
+        json_data = json.loads(response)['data_object']
+        file_url = first(json_data['urls'])['url']
+        file_name = json_data['name']
+        response = self.check_endpoint_is_working(file_url, '')
+        # Check signature of FASTQ file.
```

Comment:
This comment doesn't really apply. We don't just check the signature of the FASTQ. We open it.